### PR TITLE
We could edit the value the of product if modal is popped up again

### PR DESCRIPTION
**Pull request by Prakort**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,4 @@ config.js
 #firebase access
 serviceAccount.json
 firebase.js
+src/firebase_api_key.js
```
Comment (JameelJiwani) [inline .gitignore:32]: nice!

File: src/components/CollectionInfoContent.js [hunk 2]
```diff
@@ -2,11 +2,12 @@ import React, { useState } from "react";
...
 import { Row, Col } from "react-flexbox-grid";
 import styled from "styled-components";
-import FaceMask from "../Icons/FaceMask";
-import Gloves from "../Icons/Gloves";
-import HandSanitizer from "../Icons/HandSanitizer";
-import Suit from "../Icons/Suit";
+import { ReactComponent as FaceMask } from "../Icons/FaceMask.svg";
+import { ReactComponent as Gloves } from "../Icons/Gloves.svg";
+import { ReactComponent as Suit } from "../Icons/Suit.svg";
+import { ReactComponent as HandSanitizer} from '../Icons/disinfectant.svg';
```
Comment (JameelJiwani) [inline src/components/CollectionInfoContent.js:8]: Is there a reason why you made this change?

Having SVG's as React components gives opportunities to allow props to be passed into them

File: src/components/CollectionInfoContent.js [hunk 3]
```diff
@@ -92,20 +93,85 @@ const FlexForm = styled(Form)`
   margin: auto;
 `;
 
+const ModalCustom = subscribe()((props) => {
```
Comment (JameelJiwani) [inline src/components/CollectionInfoContent.js]: Can we move this into its own file?

**Review by JameelJiwani — COMMENTED** [review]
Good work 👍

I just have a few questions about the things you've changed

File: src/components/CollectionInfoContent.js (hunk 3 above)
Comment (Prakort) [reply src/components/CollectionInfoContent.js]: sure !!

File: src/components/CollectionInfoContent.js (hunk 2 above)
Comment (Prakort) [reply src/components/CollectionInfoContent.js:8]: React cannot resolve style property "enable-background" in svg. It is still an [open ticket](https://github.com/reactjs/react-magic/issues/63)

Comment (JameelJiwani) [reply src/components/CollectionInfoContent.js:8]: Good catch! We can revisit this when/if we decide to use props

**Review by JameelJiwani — APPROVED** [review]
👍

**Pull request merged**